wrappers/python/openmm/app/amberprmtopfile.py

```diff
@@ -288,21 +288,21 @@ def createSystem(self, nonbondedMethod=ff.NoCutoff, nonbondedCutoff=1.0*u.nanome
                     sys.setParticleMass(atom2.index, hydrogenMass)
                     sys.setParticleMass(atom1.index, sys.getParticleMass(atom1.index)-transferMass)
         for force in sys.getForces():
-            if isinstance(force, mm.NonbondedForce):
+            if isinstance(force, (mm.NonbondedForce, mm.CustomNonbondedForce)):
                 force.setEwaldErrorTolerance(ewaldErrorTolerance)
```

Comment:
CustomNonbondedForce doesn't have a `setEwaldErrorTolerance()` method.  That's why we need separate checks for that (which should only be called on NonbondedForce) and for the switching function (which should be called on either of them).